packages/react-components/src/stories/RadioButton.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { ComponentMeta } from '@storybook/react';
+
+import {
+  RadioButton as RadioButtonComponent,
+  IRadioButtonProps,
+} from '../components/RadioButton';
+
+export default {
+  title: 'Components/RadioButton',
+  component: RadioButtonComponent,
+  argTypes: { onChange: { action: 'changed' } },
+} as ComponentMeta<typeof RadioButtonComponent>;
+
+export const RadioButton = ({
+  children,
+  ...args
+}: IRadioButtonProps): React.ReactElement => (
+  <div>
+    <RadioButtonComponent {...args}>{children}</RadioButtonComponent>
+  </div>
+);
+
+RadioButton.args = {
+  checked: false,
+  disabled: false,
+  description: 'Lorem ipsum',
+  children: 'Radio button label',
+};
```

Comment:
by casting this object literal to a props type you can gain additional type safety for actually setting those initial args values
```suggestion
} as IRadioButtonProps;
```